Re-ordering the elements on the research page

clcoffey opened the pull request:
This closes https://github.com/InnerSourceCommons/innersourcecommons.org/issues/243

tsadler1988 commented:
It would be great if we could include Russian translations for navigation text ("Back to Learning Path homepage", "Next", "Previous", "Thank you to this article's contributors!") as it's currently falling back to the default English.

@voborgus, or any other Russian speakers, could you please provide translations and I can include them in this or a future PR?

tsadler1988 commented:
Superseded by #254

clcoffey committed:
Re-ordering the elements on the research page

This closes https://github.com/InnerSourceCommons/innersourcecommons.org/issues/243
